app/src/ui/app.tsx

```diff
@@ -123,6 +123,8 @@ import { CreateTag } from './create-tag'
 import { DeleteTag } from './delete-tag'
 import { ChooseForkSettings } from './choose-fork-settings'
 import { DiscardSelection } from './discard-changes/discard-selection-dialog'
+import { LocalChangesOverwrittenDialog } from './local-changes-overwritten/local-changes-overwritten-dialog'
+import * as memoizeOne from 'memoize-one'
```

Comment:
I'm using `import * from memoizeOne` here to overcome the limitation that jest uses the commonJS version of npm dependencies, which in most cases (like `memoize-one`, `classnames` or `moment`) don't expose a default attribute to make it interoperable with default imports.

While this technically is not ES module compliant, it's a quick fix that's been already widely used in this codebase (for `moment` and `classnames`).

I'm going to create a follow-up PR to fix these in a way that makes our codebase compatible with ES6 module syntax.

Reply:
Looks like the star-import trick didn't quite do the trick here (pun intended) 😢 
> ```
> [at-loader] ./app/src/ui/app.tsx:196:35 
>    TS2349: This expression is not callable.
>  Type 'typeof import("/Users/distiller/Desktop/desktop/node_modules/@types/memoize-one/index")' >  has no call signatures.
> ```

Reply:
Damn it! It looks like the TypeScript definitions for `memoize-one` are stricter than for the other libraries and don't allow this trick.

I'm going to remove the memoization from this module for now and create a follow-up PR to try to fix the madness when importing npm packages that expose both commonJS and ES6 modules